common/flavorization/wc_title_holders.txt

```diff
@@ -84,227 +70,2398 @@ high_priestess_kaldorei_female = {
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+chieftess_female_wildhammer_count = {
+    type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+chiefdom_wildhammer_county = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+	faiths = { shamanism }
+}
+thane_male_wildhammer_duke = {
+    type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
+thane_female_wildhammer_duke = {
+    type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
+high_chiefdom_wildhammer_duchy = {
+	type = title
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
+high_thane_male_wildhammer_king = {
+    type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
+high_thane_female_wildhammer_king = {
+    type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = @kingdom_cultural_prio
+	top_liege = no
+	governments = { feudal_government tribal_government clan_government }
+	cultures = { wildhammer }
+}
+
+### Dark Iron ###
+# Feudal
+lord_male_darkiron_count = {
+    type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government clan_government }
+	cultures = { dark_iron }
+}
+lady_female_darkiron_count = {
+    type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	top_liege = no
+	governments = { feudal_government clan_government }
+	cultures = { dark_iron }
+}
+lordship_darkiron_county = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government clan_government }
+	cultures = { dark_iron }
+}
+thane_male_darkiron_duke = {
+    type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = @duchy_cultural_prio
+	top_liege = no
+	governments = { feudal_government clan_government }
+	cultures = { dark_iron }
+}
+thane_female_darkiron_duke = {
```

Comment:
Why the title not called Thanedom?

Reply:
Nope, they are high chiefdom like wildhammer. But wrong syntax so i fixed it in 2989874f8